libraries/pi4j-library-linuxfs/src/main/java/module-info.java

```diff
@@ -32,6 +32,9 @@
     // PI4J
     requires com.pi4j;
 
+    // JNA
+    requires  com.sun.jna;
```

Comment:
please remove this, if not necessary

Reply:
Doh !     And I commented that it was a remnant from my jni investigation then i failed to remove it.        Will clean this up. Thanks